### ISSUE TITLE
Netlify ESLint Check fails to complete

### ISSUE THREAD

**Issue by ann-kilzer** [WWCodeTokyo/speak-her-db#181]
I notice on most PRs, the Netlify ESLint check never completes, which means I have to use my admin privileges to make the merge.

Can we either fix it or disable it, since it's redundant with the GitHub actions eslint check?

**Comment by ann-kilzer** [comment]
I think a downside of using my admin powers to merge requests is I end up taking credit for the final merge in commit history. Then our contributors don't get the same credit for their help 😞

**Comment by tuttiq** [comment]
It seems the branches that are pending are only the ones submitted by external contributors of this project (that have to fork and submit the PR from their repo). Maybe it's a problem with the Netlify access from their branches?

**Comment by tuttiq** [comment]
Yeah, merging the PR above did't do the trick yet 😞 I wonder if there's any repo config to allow external branches to run certain actions or something like this...

<img width="949" alt="Screen Shot 2020-10-11 at 11 05 49 PM" src="https://user-images.githubusercontent.com/1096046/95680712-90051c80-0c16-11eb-943e-00132cedcca2.png">
